fem/bilininteg_ext.hpp

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2010, Lawrence Livermore National Security, LLC. Produced at
+// the Lawrence Livermore National Laboratory. LLNL-CODE-443211. All Rights
+// reserved. See file COPYRIGHT for details.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability see http://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the GNU Lesser General Public License (as published by the Free
+// Software Foundation) version 2.1 dated February 1999.
+
+#ifndef MFEM_BILININTEG_EXT
+#define MFEM_BILININTEG_EXT
+
+namespace mfem
+{
+
+/// GeometryExtension
+class GeometryExtension
+{
+public:
+   Array<int> eMap;
+   Array<double> nodes;
+   Array<double> X, J, invJ, detJ;
```

Comment:
I think we should really have a way to specify what we want to store in the future. I don't think we want to store the `invJ` ever, the `detJ` should be stored only if we don't store `J`, and `X` should be stored only if physical coordinates are going to be used, but this can wait next release.